src/App.jsx

```diff
@@ -1,15 +1,29 @@
 import React from 'react';
 import './App.scss';
+import  Sum  from './components/Sum';
 
 const App = () => (
   <>
-    <p>Sum of 2 and 3 is 5</p>
-    <p>Sum of -5 and 5 is 0</p>
-    <p>Sum of 10 and 0 is 10</p>
-    <p>Sum of 0 and 5 is 5</p>
-    <p>Sum of 0 and 0 is 0</p>
-    {/* Replace paragraphs with Sum componets */}
-    {/* And remove comment lines :) */}
+    <Sum
+      a={2}
+      b={3}
+    />
+    <Sum
+      a={-5}
+      b={5}
+    />
+    <Sum
+      a={10}
+      b={0}
+    />
+    <Sum
+      a={0}
```

Comment:
We should skip zero arguments according to the task description.
![image](https://user-images.githubusercontent.com/29862525/140043389-f6da1bff-a174-4015-bc6a-1a18d483cc68.png)

Reply:
Yes, skip these 0 arguments.
It should be handled by defaultProps.